src/SprykerEco/Zed/Heidelpay/Business/Adapter/Mapper/BasketRequestToHeidelpay.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+namespace SprykerEco\Zed\Heidelpay\Business\Adapter\Mapper;
+
+use Generated\Shared\Transfer\HeidelpayBasketRequestTransfer;
+use Heidelpay\PhpBasketApi\Object\Basket;
+use Heidelpay\PhpBasketApi\Object\BasketItem;
+
+class BasketRequestToHeidelpay implements BasketRequestToHeidelpayInterface
+{
+    public const BASKET_ITEM_GOODS_TYPE = 'goods';
```

Comment:
I bet you will use this constant in tests, so please move it to Shared

Reply:
This one is necessary only for this particular API class